src/ViewHelpers/Format/SplitViewHelper.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file belongs to the package "TYPO3 Fluid".
+ * See LICENSE.txt that was shipped with this package.
+ */
+
+namespace TYPO3Fluid\Fluid\ViewHelpers\Format;
+
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+use TYPO3Fluid\Fluid\Core\ViewHelper\Traits\CompileWithRenderStatic;
+
+final class SplitViewHelper extends AbstractViewHelper
+{
+    use CompileWithRenderStatic;
+
+    /**
+     * @var bool
+     */
+    protected $escapeChildren = false;
+
+    public function initializeArguments(): void
+    {
+        $this->registerArgument('delimiter', 'mixed', 'Delimiter string to explode with', true);
+        $this->registerArgument('value', 'string', 'The string to explode');
+        $this->registerArgument('limit', 'int', 'If limit is positive, a maximum of $limit items will be returned. If limit is negative, all items except for the last $limit items will be returned. 0 will be treated as 1.', false, PHP_INT_MAX);
+        $this->registerArgument('trim', 'bool', 'Trim items in array', false, false);
+        $this->registerArgument('removeEmpty', 'bool', 'Remove empty items from array. Items containing only whitespace are considered empty as well.', false, false);
+        $this->registerArgument('convertToIntegers', 'bool', 'Convert items to integers', false, false);
```

Comment:
No need to overcomplicate stuff, there is no need for a splitter to convert something.

Reply:
On this one I agree, I don't directly see the advantage of integer casting in a templating language that tries to hide types from the integrator. If you have for example a list of UIDs to pass to the `<f:image>` viewHelper as an image ID, this would get cast to an integer by that viewHelper.

@s2b Did you maybe have something specific in mind that could benefit from this?